group_vars/mgrs.yml.sample

```diff
@@ -7,6 +7,12 @@
 # file as a good configuration file when no variable in it.
 dummy:
 
+###########
+# MODULES #
```

Comment:
please run `generate_group_vars_sample.sh`, instead of writing it. Thanks.

Reply:
I don't understand. I ran `generate_group_vars_sample.sh` before commiting.
I re-run it , nothing new to commit, the generated file is the same.

Reply:
My bad.